test/jest/LambertCoordinaat.test.js

```diff
@@ -0,0 +1,191 @@
+import LambertCoordinaat from '../../src/lambert-coordinaat';
+
+describe('Lambert Coördinaat', () => {
+  test('of - een vrije tekst', () => {
+    const result = LambertCoordinaat.of('test');
+
+    expect(result).toBeUndefined();
+  });
+
+  test('of - "null"', () => {
+    const result = LambertCoordinaat.of(null);
+
+    expect(result).toBeUndefined();
+  });
+
+  test('of - "undefined"', () => {
+    const result = LambertCoordinaat.of(undefined);
+
+    expect(result).toBeUndefined();
+  });
+
+  test('of - een lege string', () => {
+    const result = LambertCoordinaat.of('');
+
+    expect(result).toBeUndefined();
+  });
+
+  test('of - een geldige coördinaat', () => {
+    const lambertCoordinaat = LambertCoordinaat.of('123456.78, 345232.04');
+
+    expect(lambertCoordinaat).not.toBeNull();
+    expect(lambertCoordinaat.x).toBe(123456.78);
+    expect(lambertCoordinaat.y).toBe(345232.04);
+    expect(lambertCoordinaat.toString()).toBe('123456.78, 345232.04');
```

Comment:
Is er geen 'deepEquals' of zo die ge kunt gebruiken ipv alles individueel hier af te checken?